src/bundle/Resources/public/js/scripts/admin.trash.js

```diff
@@ -0,0 +1,35 @@
+(function(global, doc) {
+    const form = doc.querySelector('form[name="location_trash"]');
+    const submitButton = form.querySelector('button[type="submit"]');
+    const allOptions = form.querySelectorAll('.ez-modal__trash-option');
+    const confirmCheckbox = form.querySelector('input[name="location_trash[confirm][]"]');
+
+    const enableButton = (button) => {
+        button.disabled = false;
+        button.classList.remove('disabled');
+    };
+
+    const disableButton = (button) => {
+        button.disabled = true;
+        button.classList.add('disabled');
+    };
+
+    if (!confirmCheckbox) {
+        enableButton(submitButton);
+
+        return;
+    }
+
+    const toggleSubmitButton = () => {
+        const areAllOptionsChecked = [...allOptions].every((option) => {
+            const inputs = [...option.querySelectorAll('input'), confirmCheckbox];
```

Comment:
For every option, you will check if `confirmCheckbox` is checked it's redundant.

Reply:
I have moved check outside this function.